examples/hc.conf

```diff
@@ -8,7 +8,7 @@ host3
 # Do a sanity check before proceding
 [script1]
 action=execute
-file=/usr/share/gdeploy/scripts/grafton-sanity-check.sh -d disk1,disk2 -h host1,host2
+file=/usr/share/ansible/gdeploy/scripts/grafton-sanity-check.sh -d disk1,disk2 -h host1,host2
```

Comment:
@dpmramesh In master it has to be: /usr/share/ansible/gdeploy/scripts/grafton-sanity-check.sh

In 2.0.1 it is /usr/share/ansible/gdeploy that is going to change in 2.0.2

We are moving out of ansible directory.